.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "cnmem"]
+	path = cnmem
+	url = https://github.com/NVIDIA/cnmem.git
```

Comment:
Hi Soumith,

No I am not sure - this was my best guess as well :)
Thanks for the input - we'll give it some more thought how to introduce additional packages to caffe.
Also, for cnmem in particular, I am going to try using alternative pool allocator via CUB's CachingDeviceAllocator.
That source comes with cudnn_v4, as part of Thrust library, hence no need to pull in extra packages.
Please let me know if you had any experience with it!

Best,
-Boris.

From: Soumith Chintala <notifications@github.com<mailto:notifications@github.com>>
Reply-To: drnikolaev/caffe <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, October 9, 2015 at 6:20 PM
To: NVIDIA/caffe <caffe@noreply.github.com<mailto:caffe@noreply.github.com>>
Cc: Boris Fomitchev <bfomitchev@nvidia.com<mailto:bfomitchev@nvidia.com>>
Subject: Re: [caffe] Caffe 0.14 cnmem (#1)


In .gitmodules<https://github.com/drnikolaev/caffe/pull/1#discussion_r41691078>:

> @@ -0,0 +1,3 @@
> +[submodule "cnmem"]
> +     path = cnmem
> +     url = https://github.com/NVIDIA/cnmem.git


@borisfom<https://github.com/borisfom> are we sure we want to go with a submodule? It took me a little while to figure out how to sync this code, and I like to think of myself as more git-savvy than most. I know that people in torch-land are comfortable with submodules, but this is new territory for caffe.

That being said, I can't think of any better solutions.

-
Reply to this email directly or view it on GitHub<https://github.com/drnikolaev/caffe/pull/1/files#r41691078>.

-----------------------------------------------------------------------------------
This email message is for the sole use of the intended recipient(s) and may contain
confidential information.  Any unauthorized review, use, disclosure or distribution
is prohibited.  If you are not the intended recipient, please contact the sender by
reply email and destroy all copies of the original message.
-----------------------------------------------------------------------------------